wasp/wasp.py

```diff
@@ -595,4 +596,41 @@ def theme(self, theme_part: str) -> int:
         idx = theme_parts.index(theme_part) * 2
         return (self._theme[idx] << 8) | self._theme[idx+1]
 
+    def store_settings(self, name, value):
```

Comment:
store/get don't really pair together. Either get/set or load/store (although to be honest I'd be tempted to go for getopt/setopt since, although it is short I think it conveys enough meaning... although that might be because I'm used to the Unix getopt call).

Reply:
Agreed. I went to simply "get/set" in the latest push.